tests/06-sim-actions.c

```diff
@@ -64,7 +64,7 @@ int main(int argc, char *argv[])
 	if (rc != 0)
 		goto out;
 
-	rc = seccomp_rule_add(ctx, SCMP_ACT_KILL_PROCESS, SCMP_SYS(fstat), 0);
+	rc = seccomp_rule_add(ctx, SCMP_ACT_KILL_PROCESS, SCMP_SYS(fstatfs), 0);
```

Comment:
You'll need to make the same change, `fstat` to `fstatfs`, in tests/06-sim-actions.py to resolve the Python test failures.